spec/views/items/index.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe "items/index", type: :view do
+  before do
+    assign(:items, [create(:item), create(:item)])
+  end
+
+  it "renders a list of items" do
+    render
+    cell_selector = Rails::VERSION::STRING >= '7' ? 'div>p' : 'tr>td'
+    assert_select cell_selector, text: Regexp.new("Name".to_s), count: 2
+    assert_select cell_selector, text: Regexp.new("Category".to_s), count: 2
+    assert_select cell_selector, text: Regexp.new("Location".to_s), count: 2
+    assert_select cell_selector, text: Regexp.new("MyDescription".to_s), count: 2
+    # assert_select cell_selector, text: Regexp.new(nil.to_s), count: 2
```

Comment:
is this left-over from debugging?

Reply:
Yes, I removed the comment and also introduced a variable for the magic number `2`